src/datomic_toolbox/core.clj

```diff
@@ -1,20 +1,31 @@
 (ns datomic-toolbox.core
   (:require [clojure.java.io :as io]
             [clojure.walk :as walk]
-            [datomic.api :as d]
-            [turbovote.resource-config :refer [config]])
+            [datomic.api :as d])
   (:refer-clojure :exclude [partition]))
 
+(def default-uri (atom nil))
+(def default-connection (atom nil))
+(def default-partition (atom nil))
+
+(defn configure! [{:keys [uri partition]}]
+  (reset! default-uri uri)
+  (reset! default-partition partition))
+
+(defn uri [] @default-uri)
```

Comment:
I see your point, but I needed an equivalent fn for `partition` for backwards compatibility, so this just felt consistent. More importantly, it encapsulated the atom as an implementation detail of how we store URI state rather than spreading it all over the code.